atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleDdlConfig.java

```diff
@@ -64,6 +64,12 @@ public boolean enableShrinkOnOracleStandardEdition() {
         return false;
     }
 
+    @Value.Default
+    // SHRINK COMPACT is an operation that blocks DML, so enable with care.
+    public boolean useShrinkCompactOnOracleStandardEdition() {
```

Comment:
This is incorrect.  SHRINK SPACE COMPACT is non-blocking.  SHRINK SPACE is blocking and is what this config is used to control so we should call this `useShrinkSpaceOnOracleStandardEdition `